### ISSUE TITLE
loklak services for joomla as plugins

### ISSUE THREAD

**Issue by sudheesh001** [loklak/loklak_server#297]
Depends on: #295 

Needs:
- loklak-php
- joomla.org
- apache / web servers running PHP

Potential plugins to port : http://extensions.joomla.org/extensions/extension?searchall=twitter&filter%5Btags%5D%5B%5D=&filter%5Bcore_catid%5D=&filter%5Bincludes%5D=&filter%5Bversions%5D=&filter%5Btype%5D=&filter%5Bhasdemo%5D=&filter%5Bnewupdated%5D=&filter%5Bscore%5D=&dir=DESC&limitstart=&controller=filter&view=extension&layout=list&Itemid=145&clearorders=0&clearfilters=1

**Comment by sopu** [comment]
For portable plugins, opening a separate issue.

**Comment by sudheesh001** [comment]
@sopu Maybe just comment out the plugins in each of these, it becomes very hard to maintain multiple trackers.

**Comment by daminisatya** [comment]
The sub issues for this #414

**Comment by sudheesh001** [comment]
Repository : https://github.com/loklak/loklak_joomla_timeline

**Comment by mariobehling** [comment]
Should be discussed outside this repository if needed.
